src/pybind/mgr/dashboard/README.rst

```diff
@@ -55,6 +55,12 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support the current versions of `Chrome <https://www.google.com/chrome/>`_
```

Comment:
you mean the latest version? i do think this is reasonable for enterprise users.
i think we need to list the versions we are to support explicitly!

Reply:
I agree that we probably need to be more explicit. For starters, I'd suggest to list the browser versions that we actually work with as part of the development. We could indicate that later versions of these should also work.

In my case, that'd be "Mozilla Firefox 61.0.2 (Linux, 64-bit)" and "Chromium 68.0.3440.75 (Linux, 64-bit)".